kolibri/plugins/management/assets/src/views/class-edit-page/role-switcher.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+
+  <div class="role-switcher">
```

Comment:
may want to give this class `white-space: nowrap` to prevent wrapping on small screens:

![image](https://cloud.githubusercontent.com/assets/2367265/24636424/ee936722-188e-11e7-801e-2c1c28f6a776.png)